packages/core/src/theme/theme.ts

```diff
@@ -292,10 +292,14 @@ export const borderRadii = {
 export const maxContainerWidth = '1280px'
 
 export const shadows = {
-  sm: '0 -1px 0 0 rgba(0,0,0,0.03),0 0 1px 0 rgba(0,0,0,0.24),0 2px 1px -1px rgba(0,0,0,0.16),0 2px 4px 0 rgba(0,0,0,0.12)',
-  md: '0 -1px 0 0 rgba(0,0,0,0.03),0 0 2px 0 rgba(0,0,0,0.2),0 4px 2px -2px rgba(0,0,0,0.12),0 4px 8px -1px rgba(0,0,0,0.16)',
-  lg: '0 -1px 0 0 rgba(0,0,0,0.03),0 1px 4px 0 rgba(0,0,0,0.2),0 6px 4px -4px rgba(0,0,0,0.12),0 8px 16px -1px rgba(0,0,0,0.16)',
-  xl: '0 -1px 0 0 rgba(0,0,0,0.03),0 2px 8px 0 rgba(0,0,0,0.16),0 10px 8px -5px rgba(0,0,0,0.16),0 12px 32px -2px rgba(0,0,0,0.16)',
+  sm:
+    '0 -1px 0 0 rgba(0,0,0,0.03),0 0 1px 0 rgba(0,0,0,0.24),0 2px 1px -1px rgba(0,0,0,0.16),0 2px 4px 0 rgba(0,0,0,0.12)',
+  md:
+    '0 -1px 0 0 rgba(0,0,0,0.03),0 0 2px 0 rgba(0,0,0,0.2),0 4px 2px -2px rgba(0,0,0,0.12),0 4px 8px -1px rgba(0,0,0,0.16)',
+  lg:
+    '0 -1px 0 0 rgba(0,0,0,0.03),0 1px 4px 0 rgba(0,0,0,0.2),0 6px 4px -4px rgba(0,0,0,0.12),0 8px 16px -1px rgba(0,0,0,0.16)',
+  xl:
+    '0 -1px 0 0 rgba(0,0,0,0.03),0 2px 8px 0 rgba(0,0,0,0.16),0 10px 8px -5px rgba(0,0,0,0.16),0 12px 32px -2px rgba(0,0,0,0.16)',
```

Comment:
It seems like there are some weird formatting changes in a few files. Are the diffs caused by the pre-commit hook?

Reply:
Yea, any of the formatting changes we're done automatically.

Reply:
Ok I was able to resolve these unexpected styling changes.  I was able to recreate locally by pulling files with those changes from main locally.  Then I ran `common/autoinstallers/rush-prettier/node_modules/.bin/pretty-quick --staged` (the script ran by the precommit hook i.e `rush-prettier`).  I then deleted my `common/autoinstallers/rush-prettier/node_modules`, then ran `rush update-autoinstaller --name rush-prettier`, which updated my node-modules and the `common/autoinstallers/rush-prettier/pnpm-lock.yaml` file.  There were diffs in the lock files in packages `resolution.integrity`, so I checked out the version from main to overwrite this.  I'm guessing it was due to my node-modules  having a different version of prettier or pertty-quick.